catalyst-gateway/event-db/migrations/V1__config_tables.sql

```diff
@@ -95,17 +95,16 @@ Defined  Data Formats:
 COMMENT ON COLUMN config.row_id IS
 'Synthetic unique key. 
 Always lookup using `cardano.follower.preview`';
-COMMENT ON COLUMN config.id IS
-'The name/id of the general config value/variable';
+COMMENT ON COLUMN config.id1 IS
+'The name/id1 of the general config value/variable';
```

Comment:
```suggestion
'The name/id1 of the general config value/variable';
'The primary id of the general config value/variable';
```

Reply:
Instead of 2nd and 3rd in the comments below, change to "secondary" and "tertiary"